<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ERP系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayoutMain">
    <item>
     <widget class="QFrame" name="sidebarFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutSidebar">
       <item>
        <widget class="QLabel" name="currentUserLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>当前用户：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="registerButton">
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logoutButton">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>注销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="switchUserButton">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>切换用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="line">
         <property name="frameShape">
          <enum>QFrame::Shape::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="materialButton">
         <property name="text">
          <string>物料管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="defectiveButton">
         <property name="text">
          <string>不良品管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bomButton">
         <property name="text">
          <string>BOM管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="orderButton">
         <property name="text">
          <string>订单管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="supplierButton">
         <property name="text">
          <string>供应商管理</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="materialPage">
       <layout class="QVBoxLayout" name="verticalLayoutMaterial">
        <item>
         <layout class="QHBoxLayout" name="materialSearchLayout">
          <item>
           <widget class="QLineEdit" name="materialSearchLineEdit">
            <property name="placeholderText">
             <string>搜索物料...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportButton">
            <property name="text">
             <string>导出Excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="materialTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="materialButtonLayout">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>添加物料</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editButton">
            <property name="text">
             <string>编辑物料</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>删除物料</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="text">
             <string>撤销</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="redoButton">
            <property name="text">
             <string>重做</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewDrawingButton">
            <property name="text">
             <string>查看图纸</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewPhotoButton">
            <property name="text">
             <string>查看实物照片</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="previewLabel">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <property name="text">
           <string>预览图像</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="defectivePage">
       <layout class="QVBoxLayout" name="verticalLayoutDefective">
        <item>
         <layout class="QHBoxLayout" name="defectiveSearchLayout">
          <item>
           <widget class="QLineEdit" name="defectiveSearchLineEdit">
            <property name="placeholderText">
             <string>搜索不良品...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportDefectiveButton">
            <property name="text">
             <string>导出Excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="defectiveTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="defectiveButtonLayout">
          <item>
           <widget class="QPushButton" name="addDefectiveButton">
            <property name="text">
             <string>添加不良品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editDefectiveButton">
            <property name="text">
             <string>编辑不良品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteDefectiveButton">
            <property name="text">
             <string>删除不良品</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bomPage">
       <layout class="QVBoxLayout" name="verticalLayoutBOM">
        <item>
         <layout class="QHBoxLayout" name="bomSearchLayout">
          <item>
           <widget class="QLineEdit" name="bomSearchLineEdit">
            <property name="placeholderText">
             <string>搜索 BOM...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportBOMButton">
            <property name="text">
             <string>导出Excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="bomTableView"/>
        </item>
        <item>
         <widget class="QTableView" name="bomMaterialTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="bomButtonLayout">
          <item>
           <widget class="QPushButton" name="addBOMButton">
            <property name="text">
             <string>添加 BOM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editBOMButton">
            <property name="text">
             <string>编辑 BOM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBOMButton">
            <property name="text">
             <string>删除 BOM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orderPage">
       <layout class="QVBoxLayout" name="verticalLayoutOrder">
        <item>
         <layout class="QHBoxLayout" name="orderSearchLayout">
          <item>
           <widget class="QLineEdit" name="orderSearchLineEdit">
            <property name="placeholderText">
             <string>搜索订单...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportOrderButton">
            <property name="text">
             <string>导出Excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="orderTabWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="orderButtonLayout">
          <item>
           <widget class="QPushButton" name="addOrderButton">
            <property name="text">
             <string>添加订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editOrderButton">
            <property name="text">
             <string>编辑订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteOrderButton">
            <property name="text">
             <string>删除订单</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="supplierPage">
       <layout class="QVBoxLayout" name="verticalLayoutSupplier">
        <item>
         <layout class="QHBoxLayout" name="supplierSearchLayout">
          <item>
           <widget class="QLineEdit" name="supplierSearchLineEdit">
            <property name="placeholderText">
             <string>搜索供应商...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportSupplierButton">
            <property name="text">
             <string>导出Excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="supplierSplitter">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <widget class="QTableView" name="supplierTableView"/>
          <widget class="QTableView" name="supplierMaterialTableView"/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="supplierButtonLayout">
          <item>
           <widget class="QPushButton" name="addSupplierButton">
            <property name="text">
             <string>添加供应商</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editSupplierButton">
            <property name="text">
             <string>编辑供应商</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteSupplierButton">
            <property name="text">
             <string>删除供应商</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionRegister"/>
    <addaction name="actionLogout"/>
    <addaction name="actionSwitchUser"/>
   </widget>
   <addaction name="menuUser"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>注册</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionSwitchUser">
   <property name="text">
    <string>切换用户</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
